spec/features/visitors/sign_up_spec.rb

```diff
@@ -201,10 +125,10 @@
       expect(current_path).to eq users_otp_path
     end
 
-    it 'disables mobile 2FA after Sign Out if user has no mobile' do
+    it 'disables 2FA after Sign Out if user has no mobile' do
```

Comment:
This should be renamed to `it 'does not save the mobile and sets unconfirmed_mobile to nil if user signs out before confirming their phone'`

and the expectations should be

`expect(user.reload.mobile).to be_nil`
`expect(user.reload.unconfirmed_mobile).to be_nil`